pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -186,3 +202,18 @@ func (c *ClusterConfig) HasSufficientSubnets() error {
 
 	return nil
 }
+
+//HasClusterEndpointAccess determines if endpoint access was configured in config file or not
+func (c *ClusterConfig) HasClusterEndpointAccess() bool {
+	hasAccess := false
+	if !(c.VPC == nil || c.VPC.ClusterEndpoints == nil) {
+		hasPublicAccess := &c.VPC.ClusterEndpoints.PublicAccess != nil && *c.VPC.ClusterEndpoints.PublicAccess
+		hasPrivateAccess := &c.VPC.ClusterEndpoints.PrivateAccess != nil && *c.VPC.ClusterEndpoints.PrivateAccess
+
+		hasAccess = hasPublicAccess || hasPrivateAccess
+	} else {
```

Comment:
nit: could remove this else, put a return at end of `then` block and `return true` at end of function

Reply:
Queued up for next updates.